core/src/main/scala/com/metamx/tranquility/druid/DruidBeams.scala

```diff
@@ -76,7 +76,86 @@ import scala.language.reflectiveCalls
   */
 object DruidBeams
 {
-  private val DefaultTimestampSpec = new TimestampSpec("timestamp", "iso", null)
+  val DefaultTimestampSpec = new TimestampSpec("timestamp", "iso", null)
```

Comment:
oops, nobody, that was from a previous version of the patch… will make private again